### PR TITLE
LPD-34668 Replace switch with if in js files

### PR DESCRIPTION
**Pull request by ghost**
@gianmarcobrunialti @alessiorendina can you verify that the change from switch to if was done correctly?

### CODE REVIEW

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Review by gianmarcobrunialti — APPROVED** [review]

**Review by alessiorendina — APPROVED** [review]

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:stop

**Review by gianmarcobrunialti — APPROVED** [review]

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:forward

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/summary/Summary.js [hunk 1]
```diff
@@ -78,19 +78,17 @@ function SummaryItem(props) {
...
+	if (style === 'big') {
+		ItemVariant = SummaryItemBigVariant;
+	}
+	else if (style === 'danger') {
+		ItemVariant = SummaryItemDangerVariant;
+	}
+	else if (style === 'divider') {
+		ItemVariant = SummaryItemDividerVariant;
+	}
+	else {
```
Comment (gianmarcobrunialti) [inline modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/summary/Summary.js:90]: You could also remove the final `else` and simply initialize `ItemVariant` as follows:

```javascript
let ItemVariant = SummaryItemBase;
```

to be consistent with the approach you used in all of the other files.

**Review by gianmarcobrunialti — COMMENTED** [review]
One non-blocking comment, just out of consistency. 

Not sure if **non-exhaustive** `if/else if` with initialized variables are formally preferable against **exhaustive** `if/else if/else`.

For sure in JS the **non-exhaustive** approach saves you:

1. a couple of lines of manual writing and browser line-parsing.
2. time in browser-parsing

But with minifiers in place, 2. shouldn't be a concern.

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Review by gianmarcobrunialti — APPROVED** [review]

**Comment by gianmarcobrunialti** [comment]
ci:forward

**Comment by CrescenzoRegaLF** [comment]
ci:report:78223221

**Comment by gianmarcobrunialti** [comment]
ci:forward:force

**Pull request closed without merging**